.travis.yml

```diff
@@ -2,7 +2,7 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode7.3
```

Comment:
Maybe this has something promising in it.

https://www.mail-archive.com/lldb-dev@lists.llvm.org/msg01414.html